po/ko.po

```diff
@@ -29,755 +29,751 @@ msgid "Keyboard Shortcuts"
 msgstr "단축키"
 
 #: src/SUMMARY.md:7
-msgid "Translations"
-msgstr "다른 언어들"
```

Comment:
Is this missing?

Reply:
You probably need to generate po/messages.pot from the HEAD first.

Reply:
Done